### DISCUSSION TITLE
Using Scrollbar with angular: Init/InitAll does not exist

### DISCUSSION THREAD

**Discussion by jzhang172** [idiotWu/smooth-scrollbar#524] [General]
I am using Angular 14 currently, 
When I try using   `Scrollbar.initAll(); `in my component, I get the error: 
```

any
Property 'initAll' does not exist on type 'typeof Scrollbar'.ts(2339)
```

**Comment by sadeghbarati** [comment]
Hi, can you share your repository or reproduction link on Stackblitz or Codesandbox?

**Reply by jzhang172** [reply]
Yes, here is the link: https://stackblitz.com/edit/angular-ivy-8onqjy?file=src/app/app.component.css

**Reply by sadeghbarati** [reply]
Well because you using abstract component that property `init` does not exist on it

Use this instead

```diff
- import { Scrollbar } from 'smooth-scrollbar/scrollbar'
+ import Scrollbar from 'smooth-scrollbar'
```

**Reply by jzhang172** [reply]
> Well because you using abstract component that property `init` does not exist on it
> 
> Use this instead
> 
> ```diff
> - import { Scrollbar } from 'smooth-scrollbar/scrollbar'
> + import Scrollbar from 'smooth-scrollbar'
> ```

Thank you, it works.   Is there a way to use this on body without setting a specific height on body?

**Reply by sadeghbarati** [reply]
Based on documentation you have to set height on your element

And it's better to init scrollbar on another element other than body tag, so you can handle fixed element easily

Simply put your fixed elements outside of that scrollbar element

**Reply by jzhang172** [reply]
> Based on documentation you have to set height on your element
> 
> And it's better to init scrollbar on another element other than body tag, so you can handle fixed element easily
> 
> Simply put your fixed elements outside of that scrollbar element

Okay, thank you, makes sense!
